### PR TITLE
Fix press highlight for the resource type button

### PR DESCRIPTION
**Pull request by vadi2**
# Before
before-resourcetype-outline

# Now
fix-resourcetype-outline

### CODE REVIEW

File: src/components/ResourceTypeSelector/ResourceTypeSelector.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ const CategoryButton = ({ resourceType, selectedResourceType, selectResourceType
     === resourceType ? styles.buttonSelectedText : styles.buttonText;
 
   return (
-    <Button style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
+    <Button rounded style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
       <Text style={buttonTextStyle}>{categoryDisplay}</Text>
     </Button>
```
Comment (gilhsu) [inline src/components/ResourceTypeSelector/ResourceTypeSelector.js:25]: This isn't the best fix. I tested this below and it works:
`<TouchableOpacity style={buttonStyle} onPress={() => selectResourceTypeAction(resourceType)}>
      <Text style={buttonTextStyle}>{categoryDisplay}</Text>
    </TouchableOpacity>`
 
 Also need style changes to `button` and `buttonSelected`: add `padding: 10` and `borderRadius: 20` to both.

Comment (vadi2) [reply src/components/ResourceTypeSelector/ResourceTypeSelector.js:25]: What's wrong with it? Seems very nice and clean, and defers responsibility to the framework.

Comment (gilhsu) [reply src/components/ResourceTypeSelector/ResourceTypeSelector.js:25]: The onPress ripple area is still a different shape than the body of the button.

**Comment by vadi2** [comment]
As discussed, alternative introduces noticeable latency to the UI - this'll do for the time being.

**Pull request merged**